backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/process/impl/ADProcessDAO.java

```diff
@@ -65,7 +66,7 @@
 
 public class ADProcessDAO implements IADProcessDAO
 {
-	private static final Logger logger = LogManager.getLogger(ADProcessDAO.class);
+	private static final transient Logger logger = LogManager.getLogger(ADProcessDAO.class);
```

Comment:
why adding `transient`? nowadays, i got a warning when used

Reply:
it came from a baseline merge. Fixed now